app/views/posts/new.html.erb

```diff
@@ -1,10 +1,11 @@
-<%= t "posts.add" %>
-<%= form_for :post, url: posts_path do |f| %>
-  <%= render "shared/error_postmessages" %>
+<div class="well">
+  <h1><%= t "posts.add" %></h1>
+  <%= form_for :post, url: posts_path do |f| %>
+    <%= render "shared/error_postmessages" %>
 
-  <%= f.label :title %>
-  <%= f.text_field :title, class: "form-control" %>
-  <%= f.label :body %>
-  <%= f.text_area :body, class: "form-control" %>
-  <%= f.submit t("posts.submit"), class: "btn btn-primary" %>
-<% end %>
+    <%= f.label :title %>
+    <%= f.text_field :title, class: "form-control" %>
+    <%= f.label :body %>
+    <%= f.text_area :body, class: "form-control" %>
+    <%= f.submit t("posts.submit"), class: "btn btn-primary" %>
+  <% end %>
```

Comment:
wrong indent

Reply:
There should have indent between div right?